plugins/CoreHome/LoginAllowlist.php

```diff
@@ -20,15 +20,15 @@
  * This class is in CoreHome since some alternative Login plugins disable the Login plugin and we want to ensure the
  * feature works for all login plugins.
  */
-class LoginWhitelist
+class LoginAllowlist
 {
-    public function shouldWhitelistApplyToAPI()
+    public function shouldAllowlistApplyToAPI()
     {
         $general = $this->getGeneralConfig();
-        return !empty($general['login_whitelist_apply_to_reporting_api_requests']);
+        return !empty($general['login_allowlist_apply_to_reporting_api_requests']) || !empty($general['login_whitelist_apply_to_reporting_api_requests']);
```

Comment:
The fallback to the old name might actually not work as expected in all cases as the default value of those setting is `1`.
Lets say someone does not configure `login_allowlist_apply_to_reporting_api_requests`, but uses `login_whitelist_apply_to_reporting_api_requests = 0`. That would actually not work, as `login_allowlist_apply_to_reporting_api_requests` defaults to `1`.

I guess that might be fine, as `login_allowlist_apply_to_reporting_api_requests` will be set on update if needed, but actually I think we could remove the fallback here then...